public/assets/scss/_index.scss

```diff
@@ -461,8 +471,18 @@ h5 .btn-link {
         padding: 0.4rem;
         width: 90%;
     }
-    .about br {
-        display: none;
+    .about {
+        padding: 0 3rem;
+        &-ball {
+            padding-top: 0;
+        }
+        br {
```

Comment:
This class can be removed too and manipulated via bootstrap's display classes as explained above.